src/main/java/CoinLocker/CoinLockerSystem.java

```diff
@@ -215,13 +227,13 @@ private void openLocker() {
         } catch (CancelOpeningLockerException e) {
             System.out.println("Cancel opening a locker.");
         } catch (InvalidPasswordLimitReachedException e) {
-            System.out.println("You entered wrong password "+INVALID_PW_LIMIT+ " times. Please contact to the office.");
+            System.out.println("You entered wrong password " + INVALID_PW_LIMIT + " times. Please contact to the office.");
```

Comment:
Should we mark the locker as permanently locked to prevent the user from retrying it again from the scratch?
If you thought about it but did not want to include it in this cr, TODO comment would be nice.

Reply:
Added new list for temporarily locked lockers. Added logic and test cases related to the case.